### DISCUSSION TITLE
❌ Unable to join Bytewax Slack community

### DISCUSSION THREAD

**Discussion by manevant** [bytewax/bytewax#504] [General]
Hello! Its seems like it's unable to join a Slack community via [given link](https://join.slack.com/t/bytewaxcommunity/shared_invite/zt-1lhq9bxbr-T3CXxR_9RIUGb4qcBK26Qw)

When I'm trying to sign up via my google account, I see this 
<img width="660" alt="Screenshot 2024-08-20 at 18 45 00" src="https://github.com/user-attachments/assets/e1cffb3e-d369-4b76-96b4-a99a10190f8a">

There are any tricks or caveats? Or I'm just blockhead :)
Or there is any permission issues on slack settings side?

**Comment by awmatheson** [comment]
That is weird, where was this link from? Maybe it is the wrong link and I need to generate a new one.

I tried the link you shared and it successfully redirected me to the Bytewax community slack, but I am already authenticated so maybe there was some change on the slack side 🤷‍♂️.

**Reply by manevant** [reply]
I fount that invite link in the header at [bytewax](https://bytewax.io/). 
Maybe the link is already expire? 
According to [this slack article](https://slack.com/intl/en-gb/help/articles/201330256-invite-new-members-to-your-workspace#share-an-invitation-link)

> Each invitation link can be used by up to 400 people.

so maybe its root of the problem.

**Reply by awmatheson** [reply]
We will update that thanks for pointing it out. In the meantime, this should still work - https://join.slack.com/t/bytewaxcommunity/shared_invite/zt-vkos2f6r-_SeT9pF2~n9ArOaeI3ND2w

**Reply by manevant** [reply]
Yay! im in, thx!

**Comment by EmilReiter** [comment]
Hello, I would like to join the bytewax community on slack, but I have the exact same problem when clicking on the slack icon in the header at https://bytewax.io/ and when using the link https://join.slack.com/t/bytewaxcommunity/shared_invite/zt-vkos2f6r-_SeT9pF2~n9ArOaeI3ND2w .
